bugtool/cmd/root.go

```diff
@@ -0,0 +1,417 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/cilium/cilium/common"
+	"github.com/cilium/cilium/daemon/defaults"
+
+	"github.com/spf13/cobra"
+)
+
+// BugtoolRootCmd is the top level command for the bugtool.
+var BugtoolRootCmd = &cobra.Command{
+	Use:   "bugtool",
+	Short: "Cilium agent debugging tool",
+	Long:  "cilium-bugtool - capture system and node information for debugging a Cilium node",
+	Run: func(cmd *cobra.Command, args []string) {
+		runTool()
+	},
+}
+
+const disclaimer = `DISCLAIMER
+This tool has copied information about your environment.
+If you are going to register a issue on GitHub, please
+only provide files from the archive you have reviewed
+for sensitive information. For more information on what
+is being collected, please see[0].
+
+Thanks.
+
+[0]: http://docs.cilium.io/en/stable/troubleshooting/#single-node-bugtool
```

Comment:
If this changes over time and someone is using a new version, will that be an issue? Is there a way we can make the version of the documentation dependent upon the version of the agent currently running?

Reply:
Sorry, I don't have a good answer for this yet. Will think about it some more, but if you have any ideas please share.